THIS Event 2020/Abstract.tex

```diff
@@ -0,0 +1,18 @@
+\documentclass{article}
+
+\title{A game theoretic model of the behavioural gaming that takes place at the EMS - ED interface}
+\author{Michalis Panayides}
+
+\begin{document}
+
+\maketitle
+\begin{abstract}
+    Mathematical modelling can be utilised to model a vast range of scenarios applied to areas like healthcare, supermarkets and many more. Although, such methods are really powerful, they sometimes lack the possible insight that one may get by considering more qualitative approaches. The \textit{mother} of such approaches is considered by many to be ethnography. 
```

Comment:
Operational Research (OR) techniques provide a toolkit of mathematical modelling approaches that are routinely used for problem solving in many sectors, increasingly so in healthcare.